### PR TITLE
Misc fixes 2022-05-11

### PR DESCRIPTION
**Pull request by nsoranzo**
- Replace committer name and GitHub username. Follow up on commit 559b0d1 .
- Remove former devteam members from uncredited pull requestor list.
- Fix type annotation and simplify `scripts/bootstrap_history.py`. Use f-strings.
- Don't list pull requests when checking for blocking issues.
-  Fix typo in `StagingInterface` class name without breaking Planemo.
- Specify `asyncio_mode` value for pytest-asyncio. Fix the following warning in unit tests:
  ```
  .venv/lib/python3.7/site-packages/pytest_asyncio/plugin.py:191
    /home/runner/work/galaxy/galaxy/galaxy root/.venv/lib/python3.7/site-packages/pytest_asyncio/plugin.py:191: DeprecationWarning: The 'asyncio_mode' default value will change to 'strict' in future, please explicitly use 'asyncio_mode=strict' or 'asyncio_mode=auto' in pytest configuration file.
      config.issue_config_time_warning(LEGACY_MODE, stacklevel=2)
  ```
-  Add missing pytest-asyncio test requirement to webapps package, otherwise all unit tests in `test_request_scoped_sqlalchemy_sessions.py` are skipped.
- Fix warning in pytest collection:
  ```
  test/unit/web_framework/framework/test_framework_base.py:4
    /home/runner/work/galaxy/galaxy/galaxy root/test/unit/web_framework/framework/test_framework_base.py:4: PytestCollectionWarning: cannot collect test class 'TestWebapp' because it has a __init__ constructor
      class TestWebapp(WebApplication):
  ```
  Also:
  - Use f-string.
  - Simplify code using assert.

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

File: scripts/bootstrap_history.py
Comment (mvdbeek) [inline scripts/bootstrap_history.py]: I think we want this ? I do use this when preparing releases

Comment (nsoranzo) [reply scripts/bootstrap_history.py]: It seemed redundant with `--check-blocking-prs`, but I defer to you if you prefer this reverted.

File: doc/source/releases/19.05.rst
Comment (jdavcs) [inline doc/source/releases/19.05.rst:216]: There are so many mentions of ic4f in PR discussions and code reviews, we can't remove them all. Besides, I would prefer to keep this and the following reference, if you don't mind (it's a nice reminder). But thanks for fixing the name in the past release announcements!

Comment (nsoranzo) [reply doc/source/releases/19.05.rst:216]: @jdavcs I can restore these with the updated username, if that's OK?

If you'd like to be mentioned in the release notes for future PRs, I can remove your username from https://github.com/galaxyproject/galaxy/pull/13901/files#diff-5088adec198bd0b4206ec7080b366c9b0547edb62c3310e68b2b80f8eded6731R41 , let me know!

Comment (jdavcs) [reply doc/source/releases/19.05.rst:216]: That would be great, thank you!

Comment (nsoranzo) [reply doc/source/releases/19.05.rst:216]: Done in a rebase!

File: scripts/bootstrap_history.py
Comment (nsoranzo) [reply scripts/bootstrap_history.py]: Reverted in a rebase.

File: doc/source/releases/19.05.rst
Comment (dannon) [reply doc/source/releases/19.05.rst:216]: How do we actually want to manage that list?  As the comment states it is indeed a bit arbitrary at this point, but generally folks who are part of the core active contributing group are listed.

Comment (jdavcs) [reply doc/source/releases/19.05.rst:216]: Actually now that I think of it, since all (mostly all?) committers are not explicitly mentioned, I think it would make sense to  make it consistent across all committers. Or drop it altogether - no preference here, as long as it's consistent for everyone. @nsoranzo I'm sorry, I got confused and didn't realize that was the committer exclusion list, so if we keep that list, I'd like to be on it.

Comment (nsoranzo) [reply doc/source/releases/19.05.rst:216]: I think this PR goes in the right direction (i.e. dropping the list), but maybe that could be a specific follow up PR? Happy to open it for discussion, if you agree.

Comment (jdavcs) [reply doc/source/releases/19.05.rst:216]: Yes, of course, no need to hold this up! Let's get this in and then we can have another one.

**Review by jdavcs — APPROVED** [review]

File: doc/source/releases/19.05.rst
Comment (nsoranzo) [reply doc/source/releases/19.05.rst:216]: Thanks, I've opened https://github.com/galaxyproject/galaxy/pull/13915 .

**Pull request merged**
